freq/static/freq/js/ajaxsetup.js

```diff
@@ -52,6 +54,40 @@ $.ajaxSetup({
 });
 
 
+var spinnerCount = 0;
+
+
+function waitSpinnerShow(){
+  spinnerCount += 1;
+  if (spinnerCount === 1){
+    var baseContainer = $("#base-container");
+    var width, height;
+    if (baseContainer.length){
+      height = (baseContainer.height() - 200) * -0.55;
+      width = (baseContainer.width() - 150) * 0.49 + "px";
+    } else {
+      var map = $('#map');
+      height = map.height() * -0.55;
+      width = map.width() * 0.49 + "px";
```

Comment:
Dit gaat goed? Kennelijk wel. Ik vroeg me af of het nog naar een geheel getal afgerond moest worden.